src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -0,0 +1,122 @@
+package lambda.part1.exercise;
+
+import com.google.common.base.Predicate;
+import com.google.common.collect.FluentIterable;
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class Exercise1 {
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortComparator() {
+        Person[] persons = getPersons();
+
+        Arrays.sort(persons, Comparator.comparingInt(Person::getAge));
```

Comment:
Here you should declare `class` that implements `Comparator` and use it's instance.

Reply:
done